### PR TITLE
GCC bump to 6.2

### PR DESCRIPTION
**Pull request by bilke**

### CODE REVIEW

**Comment by endJunction** [comment]
@bilke Does it mean that Ubuntu 16.04 users can not use conan (e.g. for VTK) ?

**Comment by bilke** [comment]
@endJunction Yes, correct.

**Comment by endJunction** [comment]
@bilke Even not to build there system specific libraries?

**Comment by endJunction** [comment]
@bilke Even not to build there system specific libraries?

**Comment by endJunction** [comment]
@bilke Even not to build there system specific libraries?

**Comment by endJunction** [comment]
@bilke Even not to build there system specific libraries?

**Comment by endJunction** [comment]
@bilke Even not to build there system specific libraries?

**Comment by bilke** [comment]
@endJunction That is possible with `-DOGS_CONAN_BUILD=VTK`.

File: scripts/cmake/ConanSetup.cmake [hunk 1]
```diff
@@ -16,6 +16,12 @@ if(CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
         "using Conan. Specify CMAKE_BUILD_TYPE!")
 endif()
 
+# $ cat /etc/os-release | grep VERSION_ID
+# VERSION_ID="17.10"
+if(COMPILER_IS_GCC AND CMAKE_CXX_COMPILER_VERSION VERSION_EQUAL 6 AND UBUNTU_VERSION VERSION_EQUAL 16)
```
Comment (bilke) [inline scripts/cmake/ConanSetup.cmake]: Ups...

**Pull request merged**